config/env.sh

```diff
@@ -13,9 +13,15 @@ MASTER2=172.16.0.23   # static IP of master2 VM
 WORKER0=172.16.0.24   # static IP of worker VM
 WORKER1=172.16.0.25   # static IP of worker VM
 WORKER2=172.16.0.26   # STATIC IP of worker VM
+WORKER3=172.16.0.27   # STATIC IP of worker VM
+WORKER4=172.16.0.28   # STATIC IP of worker VM
+WORKER5=172.16.0.29   # STATIC IP of worker VM
+STORAGE0=172.16.0.30   # STATIC IP of worker VM
```

Comment:
STORAGE is optional?  can you add (optional) or something?
also "static ip of storage vm" right?

Reply:
I was thinking of taking all the cp4m files including this one and moving them to a new directory. These entries won't hurt the current 3 worker node stream but no point in confusing everyone